group/iab/IAB_Decisions.md

```diff
@@ -54,56 +59,47 @@ for the IAB chair to determine whether it will be held open or secret.
 ## Normal Process
 
 The IAB chair (or executive director on the instructions of the chair) will 
-typically follow the following process:
+typically follow this process to initiate a vote:
 
-- If the request for a vote comes from an external source (e.g., the NomCom
-  provides its IESG slate for confirmation), then, as soon as possible after
-  receiving the request, the IAB chair adds the vote topic to the agenda of the
-  next IAB teleconference. If the next meeting is less than 48 hours away, the
-  chair should instead add the topic to the following teleconference.
+- The chair emails the list of voting members to start an email discussion on the
+topic, define when the voting will begin, and optionally indicate that the ballots
```

Comment:
s/ballots/votes

Reason: May be stylistic but I think "ballot" like "the IAB decides to do X" and what we may optionally want to be secret is the specific votes (e.g., Jason voted YES).